pages/instructorRequestCourse/instructorRequestCourse.ejs

```diff
@@ -0,0 +1,91 @@
+                    </div>
+                    <div class="card-body">
+                        <%- error_message  %>
+                    </div>
+                </div>
+            <% } /* error message */ %>
+
+            <!-- Previous Course Requests -->
+            <% if (course_requests.length > 0) { %>
+                <div class="card mb-4">
+                    <div class="card-header bg-primary text-white d-flex align-items-center">
+                        Course Requests
+                    </div>
+                    <div class="table-responsive">
+                        <table class="table table-sm table-hover table-striped">
+                            <thead>
+                                <tr>
+                                    <th>Short name</th>
+                                    <th>Title</th>
+                                    <th>Status</th>
+                                    <th>Details</th>
+                            </thead>
+                            <tbody>
+                                <% course_requests.forEach(function(req, i) { %>
+                                    <tr>
+                                        <td><%= req.short_name %></td>
+                                        <td><%= req.title %></td>
+                                        <td><%- include('approvalStatusIcon', {status: req.status}); %> </td>
+                                        <td><%= req.details %></td>
+                                    </tr>
+                                <% }); %>
+                            </tbody>
+                        </table>
+                    </div>
+                </div>
+            <% } /* non-zero course requests */ %>
+
+            <!-- New Request -->
+            <div class="card mb-4">
+                <div class="card-header bg-primary text-white d-flex align-items-center">
+                    Request a New Course
+                </div>
+                <form class="question-form" name="course-request" method="POST">
+                    <div class="card-body">
+                        <% if (!has_one_course) { %>
+                            <!-- Small blurb for people that have never requested a course before -->
+                            <p>
+                                To request a course space, you need to provide the <strong>course rubric and number</strong> (for example, XC 101) and the course's <strong>title</strong>. You may also optionally provide your GitHub username if you would like the ability to edit content offline.
+                            </p>
+                        <% } else { %>
+                            <p>
+                                Your course request is pending. Once your request is reviewed by a member of staff and your course space is created, you will receive an email with further instructions and resources to get started.
+                            </p>
+                        <% } %>
```

Comment:
I suggest not bothering with all this conditional logic. I think the "pending" nature of existing requests is completely clear from the table above, and the "Request a new course" card is also super clear. Rather than describing the form in a paragraph of text, I suggest putting [help text](https://getbootstrap.com/docs/4.5/components/forms/#help-text) next to the various form inputs.